scripts/channel_map.py

```diff
@@ -2,14 +2,19 @@
 
 class ChannelMap():
     channel_map = {
+        'release/6.0': {
+            'tfm': 'net6.0',
+            'branch': '6.0.1xx',
+            'quality': 'daily'
+        },
         '6.0': {
             'tfm': 'net6.0',
-            'branch': 'main',
+            'branch': '6.0.1xx',
             'quality': 'daily'
         },
         'main': {
-            'tfm': 'net6.0',
-            'branch': 'main',
+            'tfm': 'net7.0',
+            'branch': '7.0.1xx',
```

Comment:
I'm surprised this isn't `main` now?

Reply:
From talking with Matt Mitchell they are moving away from branches that imply source, and instead are moving to channel names that imply intent. So there will be no main channel going forward.